projects/www/src/app/pages/guide/data/entity-metadata.md

```diff
@@ -155,9 +155,10 @@ export function nameAndSayingFilter(
   entities: Villain[],
   pattern: string
 ) {
-  return PropsFilterFnFactory<Villain>[('name', 'saying')](
-    entities,
-    pattern
+  return (
+    PropsFilterFnFactory <
+    Villain >
+    [('name', 'saying')](entities, pattern)
```

Comment:
can you revert this change?

Reply:
I used `no-verify` to revert the file.  But this might be a problem as linting on `main`  is broken.  This might happen on other pull requests.   Might need to add a comment to skip linting on that code block.